pom.xml

```diff
@@ -1,151 +1,209 @@
-                <!--suppress UnresolvedMavenProperty -->
-                <project.build.number>${myBuildNumber}</project.build.number>
-            </properties>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>buildnumber-maven-plugin</artifactId>
-                        <version>1.4</version>
-                        <configuration>
-                            <revisionOnScmFailure>no.scm.config.in.pom</revisionOnScmFailure>
-                        </configuration>
-                        <executions>
-                            <execution>
-                                <id>create-timestamp-id</id>
-                                <phase>validate</phase>
-                                <goals>
-                                    <goal>create-timestamp</goal>
-                                </goals>
-                                <configuration>
-                                    <timestampFormat>dd.MM.yy-HH:mm</timestampFormat>
-                                    <timestampPropertyName>myBuildNumber</timestampPropertyName>
-                                    <!-- formats the timestamp all together like 04.12.16-14:17 and puts it in the ${myBuildNumber} buildProperty -->
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
-            <id>jenkins</id>
-            <activation>
-                <property>
-                    <name>env.BUILD_NUMBER</name>
-                </property>
-            </activation>
-            <properties>
-                <!-- Get the build number from Jenkins -->
-                <project.build.number>${env.BUILD_NUMBER}</project.build.number>
-                <javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>
-            </properties>
-            <repositories>
-                <repository>
-                    <id>Bromley Massive</id>
-                    <url>http://tardisjenkins.duckdns.org:8080/plugin/repository/everything/</url>
-                </repository>
-            </repositories>
-        </profile>
-    </profiles>
     <build>
-        <defaultGoal>clean package</defaultGoal>
-        <finalName>TARDIS</finalName>
+        <directory>${project.basedir}/target</directory>
+        <sourceDirectory>${project.basedir}/src/main/java</sourceDirectory>
+        <testSourceDirectory>${project.basedir}/src/test/java</testSourceDirectory>
+        <outputDirectory>${project.basedir}/target/classes</outputDirectory>
+        <testOutputDirectory>${project.basedir}/target/test-classes</testOutputDirectory>
+        <resources>
+            <resource>
+                <!-- Use POM properties in plugin.yml -->
```

Comment:
Did you actually test this?
While the property in plugin.yml is replaced with the version number correctly, the rest of the 133 other resource files are no longer copied into the TARDIS JAR file...

Reply:
Need to add:

```
<resource>
      <directory>${project.basedir}/src/main/resources</directory>
      <filtering>false</filtering>
</resource>
```
To copy the rest of the resources.

Reply:
Alternatively,

```xml
<resource>
    <directory>${project.basedir}/src/main/resources</directory>
    <filtering>true</filtering>
    <includes>
        <include>**</include>
    </includes>
</resource>
```

Or something like that.

I can also make it include the license.

Reply:
That will make it filter every file in the project... we only want to filter plugin.yml
18f611b